packages/opentelemetry-web/test/WebTracer.test.ts

```diff
@@ -78,5 +79,59 @@ describe('WebTracer', () => {
         tracer = new WebTracer({});
       });
     });
+
+    describe('when scopeManager is "ZoneScopeManager"', () => {
+      it('should correctly return the scopes for 3 parallel actions', () => {
```

Comment:
Nice to see this!!!

Would it make sense though to make this test part of the `ZoneScopeManager` instead? And maybe have a more basic async test here?